blocks/header/header.css

```diff
@@ -0,0 +1,46 @@
+body
+{
+    width: 1673px;

```

Comment:
Imagine how this will look on tablet or mobile 🙂 

It's always better to use `max-width` instead. It will allow smaller screens to have flexible `width`.